beancount_import/source/schwab_csv.py

```diff
@@ -683,6 +703,7 @@ def process_positions(
 TAXES_ACCOUNT_KEY = "taxes_account"
 DATE_FORMAT = "%m/%d/%Y"
 TITLE_RE = re.compile(r'"Transactions  for account (?P<account>.+) as of (?P<when>.+)"')
+SYMBOL_RE = re.compile(r'[^\d\w]')
```

Comment:
Can we name this `STRIP_FROM_SYMBOL_RE` to clarify that it is not a regex to match a valid symbol name, but rather a regex to match invalid characters?

Reply:
Done.